python-frontend/src/test/java/org/sonar/python/tree/PythonTreeMakerTest.java

```diff
@@ -2084,6 +2083,24 @@ public void string_interpolation_nested_expressions_in_format_specifier() {
     assertThat(formatSpecifier.formatExpressions().get(1).expression().is(Kind.MULTIPLICATION)).isTrue();
   }
 
+  @Test
+  public void string_multiple_expressions() {
+    setRootRule(PythonGrammar.ATOM);
+    Expression exp = parse("f\"a = {h,w,a == 100,foo(),(asd, asadas)}\"", treeMaker::expression);
+    StringLiteral stringLiteral = (StringLiteral) exp;
+    assertThat(stringLiteral.stringElements()).hasSize(1);
+    StringElement elmt = stringLiteral.stringElements().get(0);
+
+    assertThat(elmt.isInterpolated()).isTrue();
+    assertThat(elmt.formattedExpressions()).hasSize(1);
+    FormattedExpression formattedExpression = elmt.formattedExpressions().get(0);
+
+    Expression expression = formattedExpression.expression();
+    assertThat(expression).isNotNull();
+    assertThat(expression.getKind()).isEqualTo(Kind.TUPLE);
+    assertThat(expression.children()).hasSize(9);
+  }
+
```

Comment:
Could you please add a test with single element tuple like`f'{x,}'`?

Reply:
done